to unbreak joining via SSH with an explicit link address.

Signed-off-by: Fabian Grünbichler <f.gruenbichler at proxmox.com>
---
 data/PVE/CLI/pvecm.pm | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/data/PVE/CLI/pvecm.pm b/data/PVE/CLI/pvecm.pm
index b381f4f..fe099d4 100755
--- /usr/share/perl5/PVE/CLI/pvecm.pm
+++ /usr/share/perl5/PVE/CLI/pvecm.pm
@@ -405,9 +405,11 @@ __PACKAGE__->register_method ({
 	    push @$cmd, '--nodeid', $param->{nodeid} if $param->{nodeid};
 	    push @$cmd, '--votes', $param->{votes} if defined($param->{votes});
 
+	    my $link_desc = get_standard_option('corosync-link');
+
 	    foreach my $link (keys %$links) {
 		push @$cmd, "--link$link", PVE::JSONSchema::print_property_string(
-		    $links->{$link}, get_standard_option('corosync-link'));
+		    $links->{$link}, $link_desc->{format});
 	    }
 
        # this will be used as fallback if no links are specified
-- 
 
